content/en/building/reference/api.md

```diff
@@ -505,6 +505,254 @@ Endpoint used by cht-gateway to send sms messages. More documentation in the [ch
+        "_id": "a048aa9d-3fd0-4182-a7c1-ed65cbf29aa0",
+        "_rev": "1-bc7ed2cf5e5997734513faab5f818c3e",
+        "type": "clinic",
+        "name": "Household",
+        "external_id": "rR9Ul",
+        "notes": "Atque non carmen patruus rem toties ceno uredo ad.\nPossimus aro victoria nam terra clamo campana.",
+        "meta": {
+            "created_by": "medic",
+            "created_by_person_uuid": "",
+            "created_by_place_uuid": ""
+        },
+        "reported_date": 1740727312910,
+        "parent": {
+            "_id": "d828971b-d796-45b5-ab1a-943622d906a1",
+            "_rev": "1-1c8e9ae54655e1892686e854e7d404ea",
+            "type": "health_center",
+            "name": "Health Center",
+            "external_id": "GlghB",
+            "notes": "Turbo amor utilis surgo vomica cedo.\nAlveus tabella tondeo itaque.",
+            "meta": {
+                "created_by": "medic",
+                "created_by_person_uuid": "",
+                "created_by_place_uuid": ""
+            },
+            "reported_date": 1742160968002,
+            "parent": {
+                "_id": "3bb8af4d-ee91-4889-9e11-5ba09a833b2e",
+                "_rev": "1-6b109ceccdfcd13f9f78968262d8c82a",
+                "type": "district_hospital",
+                "name": "Hospital",
+                "external_id": "Xge4N",
+                "notes": "Attonbitus sperno cernuus.\nVarius temeritas suadeo cimentarius tum.",
+                "meta": {
+                    "created_by": "medic",
+                    "created_by_person_uuid": "",
+                    "created_by_place_uuid": ""
+                },
+                "reported_date": 1741973509708
+            }
+        }
+    }
+}
+```
+### GET /api/v1/contact/uuid
+
+*Added in 4.18.0*
+
+Returns a JSON array of strings of UUID's of contacts based on the specified page parameters.
+
+#### Permissions
+
+`can_view_contacts`
+
+#### Query Parameters
+
+| Name     | Required | Description                                                                                                                                                                                                                                                                                                                       |
+|----------|----------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| type     | true     | ID of the contact_type for the contacts to fetch. _This field or freetext is required for this endpoint._                                                                                                                                                                                                                         |
```

Comment:
suggestion: I think instead of `This field or freetext is required for this endpoint.`, we could just add `true if freetext not provided` in the `Required` column. If I was a dev just trying to quickly scan the docs, I think that would be the easiest way for me to parse this.

Same goes for other similar places below.